### PR TITLE
Frontend post project view

### PR DESCRIPTION
**Pull request by middaysan**

### CODE REVIEW

File: app/models/project.rb [hunk 1]
```diff
@@ -6,6 +6,22 @@ class Project < ApplicationRecord
...
+
+  COST_TYPE = { price: 0, contract_price: 1, free: 2 }.freeze
+  enum cost_type: COST_TYPE
+
+  TYPE_TASK = { project_type: 0, test_type: 1 }.freeze
+  enum project_type: TYPE_TASK
+
+  protected
+
+  def valid_deadline_date?
```
Comment (RGBD) [inline app/models/project.rb]: Rename to validate_deadline_date, method with quesrltionmark usually return Boolean and don't mutate state

Comment (middaysan) [reply app/models/project.rb]: rename

File: app/controllers/projects_controller.rb [hunk 2]
```diff
@@ -49,6 +49,6 @@ def destroy
   private
 
   def project_params
-    params.require(:project).permit(:title, :info)
+    params.require(:project).permit(:title, :info, :cost, :cost_type, :project_type, :deadline, :skills)
```
Comment (middaysan) [inline app/controllers/projects_controller.rb]: skills: []

File: db/migrate/20190826092911_add_type_and_cost_to_project.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class AddTypeAndCostToProject < ActiveRecord::Migration[5.2]
+  def change
+    add_column :projects, :project_type, :integer, default: 0
```
Comment (middaysan) [inline db/migrate/20190826092911_add_type_and_cost_to_project.rb]: null: false

File: db/migrate/20190826092911_add_type_and_cost_to_project.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+    add_column :projects, :cost, :integer, default: 0
```
Comment (middaysan) [inline db/migrate/20190826092911_add_type_and_cost_to_project.rb]: null: false

File: db/migrate/20190826092911_add_type_and_cost_to_project.rb [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+    add_column :projects, :cost_type, :integer, default: 0
```
Comment (middaysan) [inline db/migrate/20190826092911_add_type_and_cost_to_project.rb]: null: false

**Pull request merged**